integration-tests/test_client_options.py

```diff
@@ -177,3 +179,43 @@ def test_check_show_results(insights_client):
         assert "OPENSSH_HARDENING_CONFIG_PERMS" in show_results.stdout
     finally:
         os.chmod("/etc/ssh/sshd_config", 0o600)
+
+
+def test_common_specs(insights_client):
+    common_specs = [
+        "insights.specs.Specs.date",
+        "insights.specs.Specs.dmidecode",
+        "insights.specs.Specs.fstab",
+        "insights.specs.Specs.hostname",
+        "insights.specs.Specs.hosts",
+        "insights.specs.Specs.installed_rpms",
+        "insights.specs.Specs.ip_addresses",
+        "insights.specs.Specs.ls_dev",
+        "insights.specs.Specs.lscpu",
+        "insights.specs.Specs.lspci",
+        "insights.specs.Specs.meminfo",
+        "insights.specs.Specs.mount",
+        "insights.specs.Specs.mountinfo",
+        "insights.specs.Specs.ps_auxcww",
+        "insights.specs.Specs.redhat_release",
+        "insights.specs.Specs.uname",
+        "insights.specs.Specs.yum_repos_d",
+    ]
+
+    insights_client.register()
+    output = insights_client.run("--no-upload")
+
+    pattern = r"Archive saved at (.+/[^/]+\.tar\.gz)$"
+    match = re.search(pattern, output.stdout, re.MULTILINE)
+    if match:
+        archive_path = match.group(1)
+        return archive_path
```

Comment:
What are you trying to achieve here? Should it be `assert` instead? Pytest is complaining about it:

> `/usr/local/lib/python3.9/site-packages/_pytest/python.py:199: PytestReturnNotNoneWarning: Expected None, but integration-tests/test_client_options.py::test_common_specs` returned `'/var/cache/insights-client/insights-insights-client-integration-tests-20240425115452.tar.gz'`, which will be an error in a future version of pytest.  Did you mean to use `assert` instead of `return`?
```

Reply:
yeah there was no need for returning the value, my bad there, solved